### ISSUE TITLE
Grabbing resizer on the main timeline next to the thread panel is very hard

### ISSUE THREAD

**Issue by luixxiul** [element-hq/element-web#22208]
### Steps to reproduce

1. Open Firefox
2. Open a room
3. Open thread panel
4. Send messages to enable scrollbar on the main timeline next to the thread panel
5. Try to grab the scrollbar

#### What did you expect?

The scroll bar should be able to be grabbed easily.

#### What happened instead?

The scrollbar is already so thin (https://github.com/vector-im/element-web/issues/22084) on Firefox and already difficult to grab, and it gets even more difficult due to a conflict with the panel width change function. Maybe only 1~2 pixel is available for grabbing.

https://user-images.githubusercontent.com/3362943/168455960-2f5a46e4-0073-44ad-be0b-b0c996b86484.mp4

### Operating system

Debian

### Browser information

Firefox ESR 99

### URL for webapp

localhost

### Application version

develop branch

### Will you send logs?

No

**Comment by turt2live** [comment]
The resizer is meant to be the same width as the one on the left panel: roughly 2-3x wider than the visual indicator.

**Comment by shoqvalue** [comment]
I just made a video of the same problem in chrome.  Since there already is one, no point in posting it. 

Grabbing the resizer on the message panel is just ridiculously hard to do. Move one pixel to the right, and the thread pane width resizer bar appears (looking very much like  a thumbbar, which is weird by itself). 

It take a few minute just to fully understand why scrolling doesn't seem to work. It does, once you are able to grab the thumbar. But it's just too hard to do.
